cli/src/pcluster/api/openapi/openapi.yaml

```diff
@@ -2309,3 +2319,11 @@ components:
       - ERROR
       title: ValidationLevel
       type: string
+  securitySchemes:
+    aws.auth.sigv4:
+      description: AWS Signature Version 4 authentication
+      in: header
+      name: Authorization
+      type: apiKey
+      x-amazon-apigateway-authtype: awsSigv4
+      x-apikeyInfoFunc: pcluster.api.controllers.security_controller_.info_from_aws.auth.sigv4
```

Comment:
I think this will break the flask app

Reply:
Hmm... this was generated. Do you know if it was removed by-hand previously? I can do so for now.

Reply:
unfortunately yes, manually removed. we could comment this out so that it results in the diff